app/src/main/java/com/gh4a/utils/CustomTabsHelper.java

```diff
@@ -26,6 +26,8 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import static androidx.browser.customtabs.CustomTabsService.ACTION_CUSTOM_TABS_CONNECTION;
```

Comment:
I usually avoid static imports. Could you please remove this one and change the calling place to use `CustomTabsService.ACTION_CUSTOM_TABS_CONNECTION'?

Reply:
Done